htdocs/survey_react.php

```diff
@@ -0,0 +1,336 @@
+<?php
```

Comment:
Links to this will break once this is converted to a public module.

Can you put the file in `htdocs/survey/(pagename)` so that when it's done as a public module it can still be routed to the right place and links that were already sent won't break? (Ideally, it wouldn't have a .php extension either, but if that's required can you check if you can just add a symlink to the .php file and use the link without the extension?)

Reply:
this would require to modify the .htaccess file since it'll expect that survey is a module based on the rules

Reply:
It doesn't require changing the .htaccess file. You can create a survey (or survey/ajax) directory in htdocs without any changes htaccess